### PR TITLE
[12.0] migrate product_multi_company

### PR DESCRIPTION
**Pull request by florian-dacosta**
Depends on https://github.com/OCA/multi-company/pull/123

### CODE REVIEW

**Comment by florian-dacosta** [comment]
This PR is now ready.
@pedrobaeza Is it possible to remove the work in progress label?

**Comment by LoisRForgeFlow** [comment]
@florian-dacosta can you rebase to include that recent merge and see if CI goes green?

**Comment by florian-dacosta** [comment]
Depends on https://github.com/OCA/multi-company/pull/154 to make the tests green

**Review by JoanSForgeFlow — APPROVED** [review]

**Comment by florian-dacosta** [comment]
Test are now green!
@LoisRForgeFlow Do you mind make a review?
@bealdav

**Comment by elvise** [comment]
hi, any good news for merge?
how can i help for this task...?

**Comment by florian-dacosta** [comment]
Hello @elvise 
A pull request needs at least 2 positive review to be merged.
This one for now has only 1.
If you can test it, review it, and if you think all is ok approve it, it will help.

@pedrobaeza @lmignon @legalsylvain 
You were involved in the first PR of this module, would you mind to review this trivial migration?

File: product_multi_company/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module sets the default company taxes for all the existing companies when
+a product is created. It also adds a button to set all the taxes from other
+companies matching them by tax code.
```
Comment (legalsylvain) [inline product_multi_company/readme/DESCRIPTION.rst]: This description doesnt seem to be the right one. Don't you think ?

**Review by legalsylvain — COMMENTED** [review]
A remark. LGTM otherwise.

File: product_multi_company/readme/DESCRIPTION.rst (hunk 1 above)
Comment (florian-dacosta) [reply product_multi_company/readme/DESCRIPTION.rst]: Indeed, it is very strange, I don't understand where this came from...
I fixed this anyway.

**Review by bealdav — APPROVED** [review]
Code review migration only, trivial 
Thanks

**Review by lmignon — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Ready to merge?

**Comment by bealdav** [comment]
yes

**Comment by florian-dacosta** [comment]
@LoisRForgeFlow 
It is indeed!

**Comment by LoisRForgeFlow** [comment]
Seems there are no active maintainers for this repo: https://github.com/orgs/OCA/teams/intercompany-maintainers/members.

@pedrobaeza @JordiBForgeFlow  could any of you help merging here?

**Comment by LoisRForgeFlow** [comment]
Thanks Pedro!

**Pull request merged**